README.md

```diff
@@ -10,40 +10,48 @@ This package is a thin wrapper around [TinyMCE](https://github.com/tinymce/tinym
 * For the TinyMCE Svelte Technical Reference, see: [TinyMCE Documentation - TinyMCE Svelte Technical Reference](https://www.tiny.cloud/docs/tinymce/7/svelte-ref/).
 * For our quick demos, check out the TinyMCE Svelte [Storybook](https://tinymce.github.io/tinymce-svelte/).
 
-
 ## Quick start
 
-### Create a Svelte App from a template
+### Create a Svelte App with Vite
 
-```
-npx degit sveltejs/template my-tiny-app
-cd my-tiny-app
+```bash
+npm create vite@5 tinymce-svelte-demo -- --template svelte-ts
+cd tinymce-svelte-demo
+npm install
 ```
 
 ### Add the Editor component
 
-Install the editor component in your project
-
-```
+```bash
 npm install @tinymce/tinymce-svelte
 ```
 
-## Import the TinyMCE component
+### Import the TinyMCE component
 
-Import the TinyMCE component inside the script tag of your Svelte app
+Replace the contents of `src/App.svelte` with the following code and replace `your-api-key` with your Tiny Cloud API key:
 
-```
+```html
 <script lang="ts">
 import Editor from '@tinymce/tinymce-svelte';
+let conf = {
+  menubar: false
```

Comment:
The Vite template has this behaviour where the `margin: 0 auto;` styling in `app.css` will cause the TinyMCE editor to move around as the user opens up the menubar because the App component size is changing. The easiest workaround I could find for this was to disable the menubar which is what TinyMCE React does to work around this problem. There might be a better solution to this.